features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Comments/CommentAdd.jsx

```diff
@@ -96,8 +96,9 @@ class CommentAdd extends React.Component {
      * @memberof CommentAdd
      */
     handleClickCancel() {
-        const { toggleShowReply } = this.props;
-        toggleShowReply();
+        this.setState({ content: '' });
+        const { handleShowReply } = this.props;
+        handleShowReply(-1);
```

Comment:
why do we need to pass a `-1` to a handler ?

Reply:
Fixed